### ISSUE TITLE
Scikit-learn now includes kappa

### ISSUE THREAD

**Issue by desilinguist** [EducationalTestingService/skll#391]
Should we just switch to it instead of computing our own? 

http://scikit-learn.org/stable/modules/generated/sklearn.metrics.cohen_kappa_score.html#sklearn.metrics.cohen_kappa_score

**Comment by aoifecahill** [comment]
I think that's a good idea :)

**Comment by desilinguist** [comment]
It supports weights options (none, linear, and quadratic) but it doesn't have a way to compute the `off_by_one` stuff that our kappa implementation allows.

**Comment by aoifecahill** [comment]
can we subclass?

**Comment by desilinguist** [comment]
Yeah, that could be a good solution. Let's look into it for the 2.0 release.

**Comment by desilinguist** [comment]
I am inclined to just remove the off-by-one kappa metrics unless someone else thinks it's particularly useful.

**Comment by desilinguist** [comment]
Let's keep this open.

**Comment by desilinguist** [comment]
Keep it open.

**Comment by desilinguist** [comment]
Keep it open.

**Comment by desilinguist** [comment]
Keep it open.

**Comment by desilinguist** [comment]
So, I looked into this. 

For their [implementation](https://github.com/scikit-learn/scikit-learn/blob/55bf5d9/sklearn/metrics/classification.py#L297), we need to explicitly list out the labels whereas in our implementation we [automatically](https://github.com/educationalTestingService/skll/blob/master/skll/metrics.py#L111) take the min and the max and figure out the full list of labels. So, if we were to switch to their implementation, we would have to write a wrapper around it that figures out the labels and passes it to their function – in order to maintain backwards compatibility. This is not as nice since we would still be maintaining code on our end. Plus their implementation does not include any off-by-one kappa computation so would lose that entirely. 

Given all of this, I am not sure this is such a useful thing to do. Perhaps if they add off-by-one computation in the future, we can revisit this.
